source/translator.d

```diff
@@ -0,0 +1,34 @@
+private string[string] defaultLanguage;
+static this()
+{
+	defaultLanguage = [
+		"run": "Run",
+		"reset": "Reset",
+		"keyboard_shortcuts": "Keyboard Shortcuts",
+	];
+}
+
+/**
+Allows per-language overwrites of UI text
+*/
+class Translator
+{
+	string[string] translations;
+
+	// in case no translation is found
+	this() {}
+
+	this(string[string] translations)
+	{
+		this.translations = translations;
+	}
+
+	// uses default language as fallback
+	string opIndex(string key) const
```

Comment:
This could potentially slowdown the rendering as its called for request for every translation key.
However, 
- we won't use many translation keys
- two dict lookups (at most) are very fast compared to all the other work done for a request
- there's a better way to do this in Vibe.d